test/it/it.test.ts

```diff
@@ -0,0 +1,181 @@
+import * as chrono from "../../src";
+import { testSingleCase, testUnexpectedResult } from "../test_util";
+import ENTimeExpressionParser from "../../src/locales/it/parsers/ITTimeExpressionParser";
+import { Meridiem } from "../../src";
+
+test("Test - Date + Time Expression", function () {
+    testSingleCase(chrono, "Qualcosa accade il 18/04/2014 13:00-16:00 appena", (result) => {
+        expect(result.text).toBe("18/04/2014 13:00-16:00");
+
+        expect(result.start).toBeDate(new Date(2014, 4 - 1, 18, 13));
+        expect(result.end).toBeDate(new Date(2014, 4 - 1, 18, 16));
+    });
+});
+
+test("Test - Time Expression", function () {
+    testSingleCase(chrono, "tra le 15:30-16:30", new Date(2020, 7 - 1, 6), (result) => {
+        expect(result.text).toBe("15:30-16:30");
+
+        expect(result.start).toBeDate(new Date(2020, 7 - 1, 6, 15, 30));
+        expect(result.end).toBeDate(new Date(2020, 7 - 1, 6, 16, 30));
+    });
+
+    testSingleCase(chrono, "4:00 GMT", new Date(2020, 7 - 1, 6), (result) => {
+        expect(result.text).toBe("4:00 GMT");
+
+        expect(result.start.get("hour")).toBe(9);
+        expect(result.start.get("minute")).toBe(0);
+        expect(result.start.get("timezoneOffset")).toBe(-480);
+    });
+});
+
+test("Test - Quoted Expressions", function () {
+    testSingleCase(chrono, "Ci incontraimo per cena (21:00 GMT)?", new Date(2020, 7 - 1, 6), (result) => {
+        expect(result.text).toContain("21:00 GMT");
+    });
+
+    testSingleCase(chrono, "intorno le '15:30-16:30'", new Date(2020, 7 - 1, 6), (result) => {
+        expect(result.text).toContain("15:30-16:30");
+
+        expect(result.start).toBeDate(new Date(2020, 7 - 1, 6, 15, 30));
+        expect(result.end).toBeDate(new Date(2020, 7 - 1, 6, 16, 30));
+    });
+
+    testSingleCase(chrono, "Il giorno è '18/04/2014'", (result) => {
+        expect(result.text).toContain("18/04/2014");
+        expect(result.start).toBeDate(new Date(2014, 4 - 1, 18, 12));
+    });
+});
+
+test("Test - Strict Mode", function () {
+    testUnexpectedResult(chrono.strict, "Martedì");
+});
+
+test("Test - Random text", function () {
+    testSingleCase(chrono, "Adam <Adam@supercalendar.com> написал(а):\nIl giorno è 02.07.2013", (result) => {
+        expect(result.text).toBe("02.07.2013");
+    });
+
+    testSingleCase(chrono, "174 1 Novembre 2001-31 Marzo 2002", (result) => {
+        expect(result.text).toBe("1 Novembre 2001-31 Marzo 2002");
+    });
```

Comment:
I would be nice if you could also add `it_month_name_little_endian.test.ts` or just `it_month_name.test.ts` for testing pattern with month name.


In addition to these "DD MMMM YYYY" and "DD MMMM YYYY ~ DD MMMM YYYY", I think we have not tested, the "DD - DD, MMMM YYYY" pattern yet.